frontend/src/components/Sidebar/Sidebar.tsx

```diff
@@ -134,6 +139,77 @@ function AddClusterButton() {
   );
 }
 
+function SidebarToggleButton() {
+  const dispatch = useDispatch();
+  // const sidebar = useTypedSelector(state => state.ui.sidebar);
+  const { isOpen, isNarrow, canExpand, isTemporary } = useSidebarInfo();
+
+  const { t } = useTranslation(['frequent']);
+  const isNarrowOnly = isNarrow && !canExpand;
+
+  if (isTemporary || isNarrowOnly) {
+    return null;
+  }
+
+  return (
+    <Box textAlign={isOpen ? 'right' : 'center'}>
+      <ActionButton
+        iconButtonProps={{
+          size: 'small',
+          disableRipple: true,
+          disableFocusRipple: true,
+        }}
+        onClick={() => {
+          dispatch(setWhetherSidebarOpen(!isOpen));
+        }}
+        icon={isOpen ? 'mdi:chevron-left-box-outline' : 'mdi:chevron-right-box-outline'}
+        description={t('frequent|Collapse Sidebar')}
+      />
+    </Box>
+  );
+}
+
+function DefaultLinkArea(props: { sidebarName: string; isOpen: boolean }) {
```

Comment:
Probably should be in a DefaultLinkArea.tsx with corresponding stories.

Reply:
Same answer about having already a sidebar story.